bcda/cclf/cclf_test.go

```diff
@@ -230,19 +230,38 @@ func (s *CCLFTestSuite) TestImportCCLF8_Invalid() {
 func (s *CCLFTestSuite) TestOrderACOs() {
 	var cclfMap = map[string]map[metadataKey][]*cclfFileMetadata{
 		"A1111": map[metadataKey][]*cclfFileMetadata{},
-		"A8765": map[metadataKey][]*cclfFileMetadata{},
+		"A4321": map[metadataKey][]*cclfFileMetadata{},
```

Comment:
Changed from A8765 to A4321, because duplicate A4321 was added to the test group db.